test/remote/ups_test.rb

```diff
@@ -256,4 +255,28 @@ def test_obtain_shipping_label_without_dimensions
     # ought to raise an exception and this test will fail.
     assert_instance_of ActiveShipping::LabelResponse, response
   end
+
+  def test_obtain_international_shipping_label
+    skip '<#<RuntimeError: Could not obtain shipping label. Invalid Access License number.>>.'
```

Comment:
This line is just me copying what is done in the previous test. I believe the intention is to skip remote tests for which there are not local credentials. Although this is also the reason the typo of passing in the password as the user id that I fixed in this pr didn't cause failed tests but rather just caused those tests to get skipped. I'm happy to remove it but it would cause tests to fail for people that don't have access to the ups api.